website/blog/2017-12-14-introducing-docusaurus.md

```diff
@@ -5,7 +5,7 @@ authorURL: http://twitter.com/JoelMarcey
 authorFBID: 611217057
 ---
 
-![](/img/slash-introducing.png)
+![introducing Slash](/img/slash-introducing.png)
```

Comment:
Nit - Title case - "Introducing Slash"

Reply:
Fixed. Sorry, had done that intentionally thinking it was the `alt` standard. I changed some other ones too to Title Case.